CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### Enhancements
+
+* [vro] Added new strategy StrategyForceLatestVersions that will force you to upload a newer or same version of artifacts, otherwise the build will fail. New property has been introduced `-Dvro.forceImportLatestVersions={{BOOLEAN}}` that is set by default to true
```

Comment:
In the release.md it says the strategy is by default to false: "It is by default set to `false`." I think it looks as if false is the actual value but could you double check and update the changelog?

Reply:
Oh... good catch 😆 should be by default to false

Reply:
Nice, that would be great as it is also backwards compatible

Reply:
Yeah, that was the main philosophy 😮‍💨 even tho I think some improvements to the way strategies work are needed if we are to add more

Reply:
Hmmm if you have specific idea for that maybe we can try to implement and release together with Anton's typescript update as major version